packages/perseus/src/util/answer-types.js

```diff
@@ -106,7 +106,7 @@ export const errors = {
  *
  */
 
-const KhanAnswerTypes: $FlowFixMe = {
+const KhanAnswerTypes = {
```

Comment:
I felt an object literal would be better than `$FlowFixMe` but I'm not sure I'm right.